About

VladimyrK opened the pull request:
Разметка и стили для мобильной версии секции about.

VladimyrK committed:
Начал разметку

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Добавляет разметку и некоторые стили для мобильника

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
добавляет кнопку

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
вешает стили на иконку кнопки

VladimyrK committed:
Добавляет паддинги секции

polinadudchenko commented on src/partials/about.html:
```diff
@@ -0,0 +1,42 @@
+<div class="container about">
```
здесь надо поставить тег section с классом about. А ниже уже div с контейнером. Это необходимо, чтобы семантически выделить секцию и молоко будет выходить за границы контейнера

polinadudchenko commented on src/partials/about.html:
```diff
@@ -0,0 +1,42 @@
+      src="./images/mobile/about/about.jpg"
+      alt="Коровка и графин с молоком"
+    />
+  </picture>
```
Надо подумать, мне кажется, что эта картинка не контентная, а просто для украшательства. Если так, то ее надо ставить через css

polinadudchenko commented on src/partials/about.html:
```diff
@@ -0,0 +1,42 @@
+  <p class="about__lead about__paragraph">
+    Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus lorem
+    eget, egestas eros. Maecenas eleifend erat at justo fringilla.
+  </p>
```
я бы просто сделала здесь класс about__paragraph. А на первый абзац навесить модификатор и к нему прописать стили. Например, about__paragraph--black

polinadudchenko commented on src/partials/about.html:
```diff
@@ -0,0 +1,42 @@
+    imperdiet placerat. Vestibulum euismod nisl suscipit ligula volutpat, a feugiat urna maximus.
+    Cras massa nibh, tincidunt.
+  </p>
+  <p class="about__text about__paragraph">Aliquam erat volutpat. Aenean accumsan.</p>
```
Здесь то же самое с модификатором

polinadudchenko commented on src/partials/about.html:
```diff
@@ -0,0 +1,42 @@
+    Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus lorem
+    eget, egestas eros. Maecenas eleifend erat at justo fringilla.
+  </p>
+  <p class="about__text about__paragraph">
```
Здесь то же самое с модификатором

polinadudchenko commented on src/sass/layout.scss/_about.scss:
```diff
@@ -0,0 +1,54 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
```
Не понятно, зачем здесь flex?

polinadudchenko commented on src/sass/layout.scss/_about.scss:
```diff
@@ -0,0 +1,54 @@
+  padding-top: 125px;
+  padding-bottom: 20px;
+
+  font-weight: 500;
```
font-weight лучше повесить на текст уже сразу

polinadudchenko commented on src/sass/layout.scss/_about.scss:
```diff
@@ -0,0 +1,54 @@
+  margin-bottom: 28px;
+
+  font-size: 12px;
+  line-height: 1.62;
```
font-size & line-height одинаковые, их ннадо закинуть в about__paragraph

polinadudchenko commented on src/sass/layout.scss/_about.scss:
```diff
@@ -0,0 +1,54 @@
+}
+.about__text {
+  font-size: 12px;
+  line-height: 1.62;
```
font-size & line-height одинаковые, их ннадо закинуть в about__paragraph

polinadudchenko commented on src/sass/layout.scss/_about.scss:
```diff
@@ -0,0 +1,54 @@
+}
+.about__lead {
+  margin-bottom: 28px;
+
```
не понятно откуда такой большой margin. Начинаем разметку с мобильной версии, там margin меньше

polinadudchenko commented on src/sass/layout.scss/_about.scss:
```diff
@@ -0,0 +1,54 @@
+  color: $secondary-text-color;
+}
+.about__text:not(:last-child) {
+  margin-bottom: 7px;
```
не поняла, где margin для последнего абзаца, может просмотрела

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Merge branch 'main' into about

VladimyrK committed:
Добавляет планшетные и десктопные стили

VladimyrK committed:
Убирает закоментирование секции контактов

VladimyrK committed:
Заменяет див на секцию

VladimyrK committed:
Добавляет секцию отдельно от контейнера

Молоко строго привязано к ширине контейнера. Чтобы отвязать его и дать ему расширяться на всю ширину экрана, при этом чтобы оно не обрезалось по краям, нужно объединить секции эту и нижнюю в одну. Тогда молоко будет разливаться и секцией ниже при расширении экрана.
